receiver/googlecloudspannerreceiver/config.go

```diff
@@ -0,0 +1,131 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudspannerreceiver
+
+import (
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+type Config struct {
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+
+	TopMetricsQueryMaxRows int       `mapstructure:"top_metrics_query_max_rows"`
+	Projects               []Project `mapstructure:"projects"`
+}
+
+type Project struct {
+	ID                string     `mapstructure:"project_id"`
+	ServiceAccountKey string     `mapstructure:"service_account_key"`
+	Instances         []Instance `mapstructure:"instances"`
+}
+
+type Instance struct {
+	ID        string     `mapstructure:"id"`
+	Name      string     `mapstructure:"name"`
+	Databases []Database `mapstructure:"databases"`
+}
+
+type Database struct {
+	Name string `mapstructure:"name"`
+}
+
+func (config *Config) Validate() error {
+	if config.CollectionInterval <= 0 {
```

Comment:
We can't allow zeros, since default value is 60seconds and it is taken from default config hen user does not specify other in config - that is how configuration in collector works. It sounds reasonable to add also check for values lower than 60 sec.

Reply:
Made values lower than 60 not allowed.